### PR TITLE
Completed prep-html-skeleton - Philip Choe

### PR DESCRIPTION
**Pull request by PhilChoe**

### CODE REVIEW

**Comment by PhilChoe** [comment]
Screenshot 2024-06-15 at 12-15-14 HTML Skeleton Exercise

File: prep-html-skeleton/index.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+
+<html>
+  <title>HTML Skeleton Exercise</title>
+
+  <body>
+    <h1>I built a proper HTML skeleton!</h1>
+  </body>
+</html>
```
Comment (shawnkost) [inline prep-html-skeleton/index.html]: It looks like you are missing a `<head>` tag. Your `<title>` should go inside of a `<head>`. 

It should look something like this

```
<!DOCTYPE html>
<html>
<head>
  <title>HTML Skeleton Exercise</title>
</head>
<body>
  <h1>I built a proper HTML skeleton!</h1>
</body>
</html>
```

**Review by shawnkost — CHANGES_REQUESTED** [review]
Just missing one thing here Philip!

File: prep-html-skeleton/index.html (hunk 1 above)
Comment (PhilChoe) [reply prep-html-skeleton/index.html]: prep html skeleton revision

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**